daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/DaffodilParseOutputStreamContentHandler.scala

```diff
@@ -33,6 +36,9 @@ class DaffodilParseOutputStreamContentHandler(out: OutputStream, pretty: Boolean
   extends ContentHandler with Indentable with XMLInfosetOutputter {
   private val writer = new OutputStreamWriter(out)
   private var prefixMapping: NamespaceBinding = null
+  private var currentElementPrefixMapping: NamespaceBinding = null
+  private var currentElementAttributes: Seq[String] = Seq()
```

Comment:
I think some documentation could be useful here to describe the different between prefixMapping, currentElemenetPrefixMapping, prefixMappingCOntextStack, etc.